odoo/tests/common.py

```diff
@@ -947,7 +947,7 @@ def _spawn_chrome(self, cmd):
                 os.dup2(stderr_replacement.fileno(), sys.stderr.fileno())
             os.execv(cmd[0], cmd)
 
-    def _chrome_start(self):
+    def _chrome_start(self, additional_chrome_cli_args=None):
```

Comment:
why not passing an empty dict by default ?
It would allow directly update the `switches` without verifying the presence of the argument.